packages/@webex/plugin-meetings/test/integration/spec/converged-space-meetings.js

```diff
@@ -0,0 +1,126 @@
+          event: 'meeting:added',
+          user: alice,
+        }]),
+      ]).then(([createdMeeting]) => {
+        assert.exists(createdMeeting);
+
+        meeting = createdMeeting;
+      }),
+    );
+
+    it('user "alice" joins the meeting', () =>
+      Promise.all([
+        testUtils.delayedPromise(alice.meeting.join({enableMultistream: true})),
+        testUtils.waitForEvents([
+          {scope: bob.webex.meetings, event: 'meeting:added', user: bob},
+          {scope: chris.webex.meetings, event: 'meeting:added', user: chris},
+        ]),
+      ]).then(() => {
+        assert.isTrue(!!alice.webex.meetings.meetingCollection.meetings[meeting.id].joinedWith);
+      }),
+    );
+
+    it('users "bob" and "chris" join the meeting', () =>
+      testUtils.waitForStateChange(alice.meeting, 'JOINED')
+        .then(() => Promise.all([
+          testUtils.waitForStateChange(bob.meeting, 'IDLE'),
+          testUtils.waitForStateChange(chris.meeting, 'IDLE'),
+        ])).then(() => Promise.all([
+          bob.meeting.join({enableMultistream: true}),
+          chris.meeting.join({enableMultistream: true}),
+        ])).then(() => Promise.all([
+          testUtils.waitForStateChange(bob.meeting, 'JOINED'),
+          testUtils.waitForStateChange(chris.meeting, 'JOINED'),
+        ])).then(() => {
+          assert.exists(bob.meeting.joinedWith);
+          assert.exists(chris.meeting.joinedWith);
+        }),
+    );
+
+    it('user "alice", "bob", and "chris" adds media', () =>
+      Promise.all([
+        testUtils.addMedia(alice, {streams: ['local']}),
+        testUtils.addMedia(bob, {streams: ['local']}),
+        testUtils.addMedia(chris, {streams: ['local']}),
+      ])
+        .then(() => {
+          assert.isTrue(alice.meeting.mediaProperties.mediaDirection.sendAudio);
+          assert.isTrue(alice.meeting.mediaProperties.mediaDirection.sendVideo);
+          assert.isTrue(alice.meeting.mediaProperties.mediaDirection.receiveAudio);
+          assert.isTrue(alice.meeting.mediaProperties.mediaDirection.receiveVideo);
+          assert.isTrue(bob.meeting.mediaProperties.mediaDirection.sendAudio);
+          assert.isTrue(bob.meeting.mediaProperties.mediaDirection.sendVideo);
+          assert.isTrue(bob.meeting.mediaProperties.mediaDirection.receiveAudio);
+          assert.isTrue(bob.meeting.mediaProperties.mediaDirection.receiveVideo);
+          assert.isTrue(chris.meeting.mediaProperties.mediaDirection.sendAudio);
+          assert.isTrue(chris.meeting.mediaProperties.mediaDirection.sendVideo);
+          assert.isTrue(chris.meeting.mediaProperties.mediaDirection.receiveAudio);
+          assert.isTrue(chris.meeting.mediaProperties.mediaDirection.receiveVideo);
+        })
+    );
+
+    it('user "alice", "bob", and "chris" should be using the "homer" roap connection service', () => {
+        assert.equal(alice.meeting.mediaProperties.webrtcMediaConnection.roapConnectionService, 'homer');
```

Comment:
homer should be a constant

Reply:
Pending the correct event name, `media:negotiated` does not trigger for any user. @marcin-bazyl could you provide the correct event here?